omod/pom.xml

```diff
@@ -121,19 +121,8 @@
 		<dependency>
 			<groupId>org.openmrs.module</groupId>
 			<artifactId>kenyaemr-api</artifactId>
-			<scope>compile</scope>
+			<scope>provided</scope>

```

Comment:
once if is set in the main pom.xml, might NOT be needed here, only groupId and artifactId suffice

Reply:
Aaahh I see..

Reply:
should be good now